packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -54,6 +70,18 @@ export const setQueryTaggingDebounce = wireCommit('setQueryTaggingDebounce');
  */
 export const setSessionDuration = wireCommit('setSessionDuration');
 
+/**
+ * Tracks the tagging of the query using a debounce which ends if the user
+ * accepts a query.
+ *
+ * @public
+ */
+export const trackTaggingWire = moduleDebounce(
```

Comment:
Rename this to just `trackWire` and use it directly in the wiring (you don't need to create another key for it).

```suggestion
export const trackWire = moduleDebounce(
```